### PR TITLE
Problem 12662

### PR DESCRIPTION
**Pull request by Gor397**

### CODE REVIEW

File: objects/objects.py [hunk 1]
```diff
@@ -226,7 +226,10 @@ def get_defaults(self, name):
     'syun_',
     {'folder': path_to_SVG / 'syun'}
 )
-
+svg_defaults.add_default(
+    'door',
+    {'folder': path_to_SVG}
+)
```
Comment (zidder) [inline objects/objects.py:232]: This is the default, no need to specify

File: scenes/enumerated/problem_12662/english_text.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+taskNumberString = "P. 12662"
+mianish_str = "single digit"
+erknish_str = "double digits"
+table_col_label1_str = "number of digits\\\\in one count"
```
Comment (zidder) [inline scenes/enumerated/problem_12662/english_text.py:4]: Can you explain the meaning of `in one count`?

File: objects/objects.py (hunk 1 above)
Comment (Gor397) [reply objects/objects.py:232]: What you mean? specify what?

File: scenes/enumerated/problem_12662/english_text.py (hunk 2 above)
Comment (Gor397) [reply scenes/enumerated/problem_12662/english_text.py:4]: It was mistake. 
`number of digits\\\\in one number` or `count of digits\\\\in one number`
Which one is right?

Comment (zidder) [reply scenes/enumerated/problem_12662/english_text.py:4]: The first one seems more correct, but I don't like the usage of the word `number` twice with different meanings. Can you try to make it better? If no, the first one seems fine.

File: objects/objects.py (hunk 1 above)
Comment (zidder) [reply objects/objects.py:232]: You don't need to add those 4 lines, it's the default already.

**Review by zidder — APPROVED** [review]

**Pull request merged**